examples/step-87/doc/results.dox

```diff
@@ -0,0 +1,269 @@
+from calculation of the distance to the closest points at the interface
+in the narrow band. It can be seen that the two functions coincide.
+Similarly, on the right, the original solution and the extrapolated
+solution from the interface is shown.
+
+<table align="center" class="doxtable">
+  <tr>
+    <td>
+        @image html step_87_ex_2_res_0.png
+    </td>
+    <td>
+        @image html step_87_ex_2_res_1.png
+    </td>
+  </tr>
+</table>
+
+<h3>Mini example 3</h3>
+
+We show a shortened version of the screen output.
+
+@verbatim
+Running: example 3
+  - creating background mesh
+  - creating immersed mesh
+time: 0
+  - compute to be tested values (immersed mesh)
+  - test values (background mesh)
+  - write data (background mesh)
+  - write mesh (immersed mesh)
+
+time: 0.01
+  - move support points (immersed mesh)
+  - compute to be tested values (immersed mesh)
+  - test values (background mesh)
+
+time: 0.02
+  - move support points (immersed mesh)
+  - compute to be tested values (immersed mesh)
+  - test values (background mesh)
+
+...
+
+time: 2
+  - move support points (immersed mesh)
+  - compute to be tested values (immersed mesh)
+  - test values (background mesh)
+  - write data (background mesh)
+  - write mesh (immersed mesh)
+@endverbatim
+
+The output visualized in Paraview looks like the following: The deformation of the
+immersed mesh by the reversible vortex flow can be seen. Due to discretization
+errors, the circular shape is not exactly reobtained at the end, illustrated in
+the right figure. The sharp nature of the surface-tension force vector, shown as
+vector plots, can be seen by its restriction to cells that are intersected by the
+immersed mesh.
+
+<table align="center" class="doxtable">
+  <tr>
+    <td>
+        @image html step_87_ex_3_force.0000.png
+    </td>
+    <td>
+        @image html step_87_ex_3_force.0010.png
+    </td>
+    <td>
+        @image html step_87_ex_3_force.0020.png
+    </td>
+  </tr>
+</table>
+
+<h3>Possibilities for extension</h3>
+
+This program highlights some of the main capabilities
+of the distributed evaluation routines in deal.II. However, there are many related
+topics worth mentioning:
+- Performing a distributed search is an expensive step. That is why we suggest
+to provide hints to Utilities::MPI::RemotePointEvaluation and to reuse Utilities::MPI::RemotePointEvaluation
+instances in the case that the communication pattern has not changed. Furthermore,
+there  are instances where no search is needed and the points are already sorted
+into the right cells. This is the case if the points are generated on the cell level
+(see step-85; CutFEM) or the points are automatically sorted into the correct (neighboring) cell
+(see step-68; PIC with Particles::ParticleHandler). Having said that, the Particles::ParticleHandler might use
+the described infrastructure to perform the initial sorting of particles into
+cells.
```

Comment:
I would be a bit more specific. The insert global particles functionnality from the ParticleHandler uses this feature.

Reply:
@peterrum could you please take this one over, since I am not a very experienced user with the particle infrastructure?

Reply:
> @peterrum could you please take this one over, since I am not a very experienced user of the particle infrastructure?

Done!